MediaBrowser.WebDashboard/dashboard-ui/strings/ar.json

```diff
@@ -1261,8 +1261,8 @@
     "LabelPlayMethodDirectPlay": "\u0627\u0644\u062a\u0634\u063a\u064a\u0644 \u0627\u0644\u0645\u0628\u0627\u0634\u0631",
     "LabelAudioCodec": "\u0645\u0642\u0637\u0639 \u0627\u0644\u0635\u0648\u062a: {0}",
     "LabelVideoCodec": "\u0645\u0642\u0637\u0639 \u0627\u0644\u0641\u064a\u062f\u064a\u0648: {0}",
-    "LabelLocalAccessUrl": "In-Home (LAN) access: {0}",
-    "LabelRemoteAccessUrl": "Remote (WAN) access: {0}",
+    "LabelLocalAccessUrl": "LAN address: {0}",
```

Comment:
what's the point of changing this?.. maybe it's better be worded like `In-home (LAN) address: {0}` and `External (WAN) address: {0}`? As not all people are network savvies, so they might not now what does "WAN" stand for.